src/content/docs/new-relic-solutions/get-started/intro-new-relic.mdx

```diff
@@ -21,24 +21,34 @@ redirects:
   - /docs/using-new-relic/welcome-new-relic
   - /docs/using-new-relic/welcome-new-relic/get-started/intro-new-relic
 ---
+import installNrQuick from 'images/install_nr_quick.jpg'
 
-import observability from 'images/observability.png'
+import grafanalogo from 'images/grafana-logo.png'
 
-New Relic is an observability platform that helps you build better software. You can bring in data from any digital source so that you can fully understand your system and how to improve it.
+import opentelemetrylogo from 'images/opentelemetrylogo.png'
 
-This short video shows twenty of the most common ways to get your data into New Relic (approx. 5:22 minutes):
+import prometheuslogo from 'images/prometheus-logo.png'
 
-<Video
-  id="G1qBDgTsMsU"
-  type="youtube"
-/>
+import img0NewRelicOneKubernetesClusterExplorer from 'images/2020-04-28-17.26.02.gif'
+
+New Relic is an observability platform that helps you build better software. You can bring in data from any digital source so that you can fully understand your system and how to improve it.
 
 With New Relic, you can:
 
 * **[Bring all your data together](#bring-your-data):** Instrument everything and import data from across your technology stack using our agents, integrations, and APIs, and access it from a single UI.
 * [**Analyze your data**](#analyze-your-data): Get all your data at your fingertips to find the root causes of problems and optimize your systems. Build dashboards and charts or use our powerful query language.
 * **[Respond to incidents quickly](#respond-faster)**: Our machine learning solution proactively detects and explains anomalies and warns you before they become problems.
 
+<img
+  title="2020-04-28-17.26.02.gif"
+  alt="New Relic - Kubernetes Cluster Explorer"
+  src={img0NewRelicOneKubernetesClusterExplorer}
+/>
+
+<figcaption>
+  **[one.newrelic.com](https://one.newrelic.com) > Kubernetes cluster explorer**: The cluster explorer is our powerful, fully visual answer to the challenges associated with running Kubernetes at a large scale.
```

Comment:
i think we might need a more general caption here, talking about how K8s UI is just one of our many features. something to put it in context of NR overall and explain why we chose to show K8s of all things.